Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1076,9 +1077,11 @@ def png(self, file, width=None, height=None,
         if width is not None and height is not None:
             if self.renWin.GetSize() != (width, height):
                 user_dims = (self.canvas.bgX, self.canvas.bgY, sz[0], sz[1])
-                self.renWin.SetSize(width, height)
+                # We need to set canvas.bgX and canvas.bgY before we do renWin.SetSize
+                # otherwise, canvas.bgX,canvas.bgY will win
```

Comment:
@aashish24 This is the backtrack that explains why this happens:
lastSize: 15, 11
size: 814, 606
  File "/home/danlipsa/src/uvcdat/testing/vcs/test_png_set_size.py", line 24, in <module>
    x.png(fnm,width=15)
  File "/home/danlipsa/build/uvcdat/install/lib/python2.7/site-packages/vcs/Canvas.py", line 4773, in png
    file, W, H, units, draw_white_background, **args)
  File "/home/danlipsa/build/uvcdat/install/lib/python2.7/site-packages/vcs/VTKPlots.py", line 1085, in png
    self.renWin.SetSize(width, height)
  File "/home/danlipsa/build/uvcdat/install/lib/python2.7/site-packages/vcs/VTKPlots.py", line 267, in configureEvent
    self.canvas.plot(*pargs, render=False, **key_args[i])
  File "/home/danlipsa/build/uvcdat/install/lib/python2.7/site-packages/vcs/Canvas.py", line 2424, in plot
    a = self.__plot(arglist, keyargs)
  File "/home/danlipsa/build/uvcdat/install/lib/python2.7/site-packages/vcs/Canvas.py", line 3747, in __plot
    returned_kargs = self.backend.plot(*arglist, **keyargs)
  File "/home/danlipsa/build/uvcdat/install/lib/python2.7/site-packages/vcs/VTKPlots.py", line 518, in plot
    self.renWin.SetSize(self.canvas.bgX, self.canvas.bgY)
  File "/home/danlipsa/build/uvcdat/install/lib/python2.7/site-packages/vcs/VTKPlots.py", line 227, in configureEvent